File: src/utils/args_parser.py
import sys


class ArgsParser:
    def parse_args(self):
        if len(sys.argv) > 1:
            if sys.argv[1] == "-o":
                return True
            elif sys.argv[1] == "-c":
                return False
            else:
                print("Invalid argument provided, closing browser...")
                sys.exit(1)
        return False

File: src/main/java/utils/ArgsParser.java
package utils;

public class ArgsParser {
    private Boolean keepBrowserOpen = false;

    public Boolean parseArgs(String[] args) {
        if (args.length > 0) {
            if ("-o".equals(args[0])) {
                keepBrowserOpen = true; // Keep open
            } else if ("-c".equals(args[0])) {
                keepBrowserOpen = false; // Close
            } else {
                System.out.println("Invalid argument. Use -o (keep open) or -c (close)");
                System.exit(1);
            }
        }

        return keepBrowserOpen; // Default: close
    }
}
